authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,186 @@
+
+## Inflation
+
+In both Polkadot and Ethereum the inflation is dynamic and defined as "minimum necessary inflation". The exact number will depend on the amount of staked coins.
+
+## Validators and staking economy
+
+### Polkadot
+
+In Polkadot a validator is a full client (>>FULL OR LIGHT?) of all chains and threads, thus very resource intensive on hard drive space and read/write ops, and on bandwidth.
+
+The assignment of a validator is periodically (>>HOW OFTEN?) reshuffled so that a single validator isn't always responsible for a single parachain. A validator earns DOT from block production (inflation) and transaction fees in a 80/20 spread. 80% of the transaction fee goes to the Polkadot Treasury. The validator gets rewarded twice - they set a fixed number of DOT to get per reward period (epoch - roughly 6 hours), and the rest is then dividied among all the nominators who have nominated this validator, and the validator, based on their respective amounts of staked DOT. It's important to note that nominators can still lose their DOT if they backed an incompetent validator - this is not delegated proof of stake where you simply signal your trust in someone, you literally unlock your DOTs for "burning by malfeasance" when nominating.
+
+The maximum annual reward for validators is estimated to be 20%, and that's with 50% of all DOTs in existence staked (currently 10 million DOTs as a target). It is likely to be much lower.
+
+A collator is a full node of a specific parachain. The collator passes data which becomes a block candidate, along with proof of the state transition so that the validator can replay and verify it. The collator depends on the local economy of the parachain - likely a local token. A parachain can also use DOTs as a native token, and a local token purely for governance or similar.
+
+Fishermen earn money for each valid report they make on some wrongdoing - the slashing amount is in part burned, in part sent to treasury, and in part given to the fisherman who reported the issue. The amounts depend on the offence and on how many other validators are in the same predicament (e.g. the offline penalties are much bigger if many validators are offline at the same time).
+
+There is no limit to the amount of stake a validator can commit, and the active validators are picked from the top most-staked validators (including nominations) once per era (24 hours). This ensures that all the top validators have approximately the same amount of staked DOTs, and that a single whale can't effectively take over because the reward per validator is the same - it's not proportional to the staked amount. Therefore, the reward per dot is less when you stake more, but the penalty per offence is greater because penalties are proportionate to staked amount. This system incentivizes nominators to nominate less popular validators, keeping the distribution balanced.
```

Comment:
```suggestion
There is no limit to the amount of stake a validator can commit, and the active validators are picked from the top most-staked validators (including nominations) once per era (24 hours). This ensures that all the top validators have approximately the same amount of staked DOTs, and that a single whale can't effectively take over because the reward per validator is the same - it's not proportional to the staked amount. Therefore, the reward per DOT is less when you stake more, but the penalty per offence is greater because penalties are proportionate to staked amount. This system incentivizes nominators to nominate less popular validators, keeping the distribution balanced.
```

Reply:
Should we also mention that a way around this (in a not-very-granular way) is running multiple validators?

Reply:
I think that's implied and unnecessary to explicitly mention.